tutorials/containers/kubeflow-tutorial/kubeflow-tutorial.md

```diff
@@ -86,7 +94,69 @@ After the installation is complete, copy the K8s configuration file locally:
 juju scp kubernetes-master/0:config ~/.kube/config
 ```
 
-## Installing Kubeflow using ksonnet
+## Create your GKE Cluster
+Duration: 8:00
+
+Skip to the next step if you're purely interested CDK.
```

Comment:
perhaps - "...if you have already set up a cluster with CDK"
in any case, it is missing an 'in'

Reply:
thanks!